src/js/common/schemaform/FormApp.jsx

```diff
@@ -40,6 +40,16 @@ class FormApp extends React.Component {
     if (window.History) {
       window.History.scrollRestoration = 'manual';
     }
+
+    // If we start in the middle of a form, redirect to the beginning
+    const currentPath = this.props.location.pathname;
+    const firstPagePath = this.props.routes[this.props.routes.length - 1].pageList[0].path;
```

Comment:
Yeah, I think the only reason we'd ever make the first "real" page in a form use `depends` is if we wanted it to depend on some pre-filled data, and that seems like an awful idea (famous last words). But! It's safer to put the comment in, so I'm on it!